unity3d/Assets/Swrve/SwrveSDK/Messaging/SwrveConversationCampaign.cs

```diff
@@ -46,8 +46,11 @@ public SwrveConversation GetConversationForEvent(string triggerEvent, IDictionar
 
         if (checkCampaignLimits (triggerEvent, payload, qaUser)) {
             SwrveLog.Log (string.Format ("[{0}] {1} matches a trigger in {2}", this, triggerEvent, Id));
-
-            return Conversation;
+            if (AreAssetsReady ()) {
```

Comment:
This is also done in GetNextMessage and by Message Center, why is it also needed here?